### PR TITLE
Importer Consolidation

### PR DESCRIPTION
**Pull request by rbxnk**
This PR lays the ground work for being able to finish up all the item inspectors. I've removed all but the in mem importer. The asset importer remains, but will go through the in mem path once the object is added to a scene. In practice these changes shouldn't really affect workflows, you just have to make the prefab yourself if you want it. All texture data will remain in the serialization, so scenes will be a bit larger, but this allows to standardize on knowing where a vpx internal asset lives.
I also tracked down why the editor was running slow when selecting table root. Turns out the editor does a serialized object update each tick, which can take a while due to the large amount of data on the table root. To work around that I moved the serialized data off to a sidecar child.

### CODE REVIEW

**Comment by freezy** [comment]
I haven't tested this yet, but it sounds useful :)

Basically your new importer gets the benefit of the memory importer (loads fast), while persisting data as asset so we can develop without having to re-import all the time? Also, it works at runtime? That'd be awesome!

**Comment by rbxnk** [comment]
Sort of, the asset importer itself won't store anything in the library for the .vpx file itself. But if you import into a scene, and then save that scene... then yeah it should load back up ok without re-importing.
And yup, we basically just route everything through what used to be called the memory importer, so it should work at runtime no problem.

File: VisualPinball.Unity/VisualPinball.Unity.Editor/Import/VpxAssetLazyImporter.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+			var obj = PrefabUtility.GetCorrespondingObjectFromSource(gameObject);
+			if (obj == null) return;
+
+			var path = AssetDatabase.GetAssetPath(obj);
+
+			GameObject tableRoot = new GameObject(obj.name);
+			var importer = tableRoot.AddComponent<VpxImporter>();
+			var table = TableLoader.LoadTable(path);
+			importer.Import(Path.GetFileName(path), table);
+#endif
```
[the remark this answers is not in the document]
Comment (rbxnk) [reply VisualPinball.Unity/VisualPinball.Unity.Editor/Import/VpxAssetLazyImporter.cs:29]: Yeah didn't realize this whole assembly was flagged for editor only, also added a comment to clarify the intent here.

**Comment by rbxnk** [comment]
Yeah so I think we're on roughly the same page here. The "working copy" is intended to be the Unity project itself. So for the image fields for example, eventually we'll have a tick box next to each of them for "use vpe/unity texture" or some such, and we'd let folks wire the texture ref just like any normal Unity workflow. In my mind that's where the VPE specific fields will come in.
For meshes and textures we could "bake" that data into the vpx file so it could actually be played in vpx, but that won't be an option for materials, so that's the point we'll need to build out the asset bundle export step. As you mentioned before I imagine we'll just put one big bundle as a binary blob on the table data itself.

**Comment by freezy** [comment]
Cheers then!

**Pull request merged**